lib/runtime2/threepio/utils/JsonlProcessor.ts

```diff
@@ -1,22 +1,12 @@
-export interface JsonObject {
-  [key: string]: any;
-}
-
 export interface ProcessedJsonLine {
   rawLine: string;
-  jsonContent?: JsonObject;
   error?: { message: string; code: string };
 }
 
-/**
- * JsonlStreamProcessor is a utility class to process JSON Lines (JSONL) format data from a stream.
- */
-export class JsonlStreamProcessor {
+export class JsonlProcessor {
```

Comment:
```suggestion
export class JSONLProcessor {
```
代码中使用全部大写的 JSONL，因为它是专有名词：JavaScript Object Notation Line